test/main/provider/helpers.test.js

```diff
@@ -48,3 +48,11 @@ describe('#getRawTx', () => {
     expect(tx.value).toBe('0x0')
   })
 })
+
+describe('#getSignedAddress', () => {
+  it('returns an error if no signed address is provided', () => {
+    getSignedAddress(null, 'some message', err => {
+      expect(err).toBeTruthy()
```

Comment:
You could verify the exact error.  Also does this need a happy path test?

Reply:
I don't think the exact error is too important, seems like it's just an old piece of code. would like to be able to change the error text and not have the test break, unless we eventually come up with a spec for it. didn't have a happy path test before but it's a good time to add one :+1: